scripts/ci-setup

```diff
@@ -27,6 +27,12 @@ do
   shift
 done
 
+# Remove env file to get a clean env
+echo "Removing old .env file"
+if [ -f .env ]; then
+    echo "Removing env file"
```

Comment:
```suggestion
    echo "Removing old .env file"
```

Reply:
Fixed in c92cbc214415e8500ed1074bc3fae47f4fa661c0 and c3ee92a083a71cac25f27618bae233676dc32e26